app/models/concerns/content_addressable_id.rb

```diff
@@ -0,0 +1,26 @@
+require 'digest'
```

Comment:
You actually don't need this

Reply:
after discussion concluded this is needed since its stdlib